gnucash/gnome/dialog-price-editor.c

```diff
@@ -192,6 +194,56 @@ price_to_gui (PriceEditDialog *pedit_dialog)
 }
 
 
+static gboolean
+pedit_dialog_replace_found_price (PriceEditDialog *pedit_dialog,
+                                  const gnc_commodity *commodity,
+                                  const gnc_commodity *currency, time64 t)
+{
+    gboolean price_found = FALSE;
+    GNCPrice *test_price =  gnc_pricedb_lookup_day_t64(pedit_dialog->price_db,
+                                commodity, currency, t);
+
+    if (test_price)
+    {
+        if (pedit_dialog->is_new) // new price
+            price_found = TRUE;
+        else // edit price
+        {
+            if (!gnc_price_equal (test_price, pedit_dialog->price))
+                price_found = TRUE;
+        }
+        gnc_price_unref (test_price);
+    }
+
+    if (price_found)
+    {
+        gint response;
+        GtkWidget *dialog;
+        gchar *message = _("You are about to replace an existing price?");
```

Comment:
Seems a bit stilted: "Replace price? You are about to replace an existing price?".
I'd think "Are you sure you want to replace the existing price?" would be more idiomatic.

Reply:
Will change text.